microsite/src/theme/customTheme.scss

```diff
@@ -6,6 +6,7 @@
   --ifm-color-primary-dark: #31a792;
   --ifm-color-primary-darker: #2e9e8a;
   --ifm-color-primary-darkest: #268271;
+  --ifm-link-decoration: underline;
```

Comment:
I don't think this is the fix we want here tbh. It doesn't make the text itself more readable, which is still important. Perhaps we could separate this one out to a separate PR so we can fix the rest quicker?

Reply:
yes, I want maintainer's view on this according to A11Y guidelines links should be underlined to make them distinguishable from surrounding text by this lighthouse passes this issue as fixed, if you don't want this any other suggestions?

Reply:
because then we have to come up with colors for contrast above specified in the lighthouse which will affect backstage core font colors.

Reply:
@Rugvip how about this as an alternative?
#__docusaurus > .main-wrapper{
  a{
    text-decoration: underline;
  }
}

Reply:
Hello, yes I think we should definitely make links underline as that way the link will have a [distinct style](https://dequeuniversity.com/rules/axe/4.8/link-in-text-block) and therefore meet the criterias. If we do not have a distinct style we would work on improving the color contrast but links should look like links so lets move forward with this change. With this said - I believe its safe to avoid underline in the navigation so we can update this to underline links thats part of the text content. 

@npiyush97 I think your suggestion above moves us closer to the goal but it also underlines links in the sidebar so if you can pinpoint the links in only the text content that should be enough!